higher_health/templates/healthcheck_questionnaire.html

```diff
@@ -155,14 +155,22 @@ <h3 class="heading-subtitle heading-subtitle--edu-label">Confirmation</h3>
   {{ campuses|json_script:"campus-data" }}
 {% endcache %}
 
+  <script type="text/javascript">
+    let other_university = "{{ other_university }}";
```

Comment:
I would prefer to get this from the existing list in the document, rather than adding an additional db call, especially since this will cause us to do a full table scan to find this.